environments/sbox.tfvars

```diff
@@ -0,0 +1,5 @@
+builtFrom       = "sds-jenkins-infrastructure"
```

Comment:
did you mean to commit this file?

all of it should have been set by pipeline?